### PR TITLE
Adjust path to output folder

### PR DESCRIPTION
**Pull request by rea1991**
The tool now creates a folder with the contact map name and, inside it, a folder named after the resolution. Before this change, it was creating just the innermost folder.

### CODE REVIEW

File: test/test_IO.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def test_overwrite_existing_folder(tmpdir):
             (test_dir / "file.txt").touch()
 
             assert not _directory_is_empty(test_dir)
-            remove_and_create_folder(test_dir)
+            remove_and_create_folder(test_dir, force=True)
             assert _directory_is_empty(test_dir)
```
Comment (robomics) [inline test/test_IO.py:58]: Can you add a test case ensuring things work as expected when `force=False`?

File: src/stripepy/IO.py [hunk 2]
```diff
@@ -25,52 +24,35 @@ class ANSI:
...
-        # confirm = input("The output folder already exists. Do you want to delete it? (y/n) ")
-        confirm = "y"
-
-        if confirm.lower() == "y":
-            # Remove the folder and its contents
-            shutil.rmtree(path)
-            print("Output folder and its contents have been deleted.")
+    if path.exists():
+        if not force:
+            raise RuntimeError(f"refusing to overwrite file {path}. Pass --force to overwrite.")
```
Comment (robomics) [inline src/stripepy/IO.py:33]: I think it is better to rewrite the error message to make it clear that it is a folder that will be deleted when `--force` is given

**Review by robomics — CHANGES_REQUESTED** [review]
Looks good!

I have just two minor comments.

File: src/stripepy/IO.py (hunk 2 above)
Comment (rea1991) [reply src/stripepy/IO.py]: Good point!

File: test/test_IO.py (hunk 1 above)
Comment (rea1991) [reply test/test_IO.py:65]: I added a few lines with proper assert.

**Review by robomics — APPROVED** [review]

**Pull request merged**